build.sh

```diff
@@ -111,6 +111,21 @@ else
     echo "Found LOCAL_COLLECTOR_PORT ${LOCAL_COLLECTOR_PORT}";
 fi
 
+# check if ufw command exists
+if [ -x "$(command -v ufw)" ]; then
+    ufw allow $LOCAL_COLLECTOR_PORT
```

Comment:
Doesn't this need super user access? It might not be a good idea to include it in `build.sh` because people usually won't want to run anything as `su.` It is better to give this as a separate instruction or a troubleshooting guide IMO.

Reply:
Has been modified in latest commit to only open connections from predetermined IP ranges of docker containers that will run the local collector component